app/src/main/java/com/example/parkingsystem_kotlin/util/DateUtils.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.parkingsystem_kotlin.util
+
+import android.app.DatePickerDialog
+import android.app.TimePickerDialog
+import android.content.Context
+import android.widget.EditText
+import com.example.parkingsystem_kotlin.R
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Locale
+
+object DateUtils {
+
+    fun showDatePickerDialog(context: Context, textField: EditText) {
+        val calendar = Calendar.getInstance()
+
+        val day: Int = calendar.get(Calendar.DAY_OF_MONTH)
+        val month: Int = calendar.get(Calendar.MONTH)
+        val year: Int = calendar.get(Calendar.YEAR)
+
+        val picker = DatePickerDialog(
+            context,
+            R.style.MySpinnerDatePickerStyle,
+            { _,
+              yearNow,
+              monthOfYear,
+              dayOfMonth ->
+                calendar.set(Calendar.YEAR, yearNow)
+                calendar.set(Calendar.MONTH, monthOfYear)
+                calendar.set(Calendar.DAY_OF_MONTH, dayOfMonth)
+                showTimePicker(textField, calendar, context)
+            },
+            year,
+            month,
+            day
+        )
+        picker.show()
```

Comment:
theres no need for this var, you can DatePickerDialog( ... ).show()

Reply:
Thanks